lib/internal/test_runner/test.js

```diff
@@ -761,7 +761,7 @@ class Suite extends Test {
     try {
       const { ctx, args } = this.getRunArgs();
       this.buildSuite = PromisePrototypeThen(
-        PromiseResolve(this.runInAsyncScope(this.fn, ctx, args)),
+        PromiseResolve(this.runInAsyncScope(this.fn, ctx, ...args)),
```

Comment:
Can we avoid the unsafe array iteration here? 🙈 this would require to use `ReflectApply` and `ArrayPrototypePushApply` I think, unless it’s easier to change the signature of `runInAsyncScope`

Reply:
Sure, good catch